sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_AnalyzeOperation.cs

```diff
@@ -55,7 +55,7 @@ public void AnalyzeOperation()
                 DisplayName = "AnalyzeOperationSample"
             };
 
-            AnalyzeBatchActionsOperation operation = client.StartAnalyzeBatchActions(batchDocuments, actions);
+            AnalyzeActionsOperation operation = client.StartAnalyzeActions(batchDocuments, actions);
```

Comment:
not sure if you want to remove the "batch" reference in the first argument for the samples.

Reply:
the `batch` in this case represents a batch of documents which is how we want to keep using that word in our library. 
I think in this case it is ok to leave as is